casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -137,16 +137,16 @@ class RuntimeManager private (val emptyStateHash: ByteString, runtimeContainer:
 
   private def newEval(terms: Seq[Deploy], runtime: Runtime, initHash: StateHash)(
       implicit scheduler: Scheduler): (StateHash, Seq[InternalProcessedDeploy]) = {
-    implicit val costAccountingAlg = CostAccountingAlg.unsafe[Task](CostAccount.zero)
 
     def doEval(terms: Seq[Deploy],
                hash: Blake2b256Hash,
                acc: Vector[InternalProcessedDeploy]): (StateHash, Vector[InternalProcessedDeploy]) =
       terms match {
         case deploy +: rem =>
           runtime.space.reset(hash)
-          val (cost, errors) = injAttempt(deploy, runtime.reducer, runtime.errorLog)
-          val newCheckpoint  = runtime.space.createCheckpoint()
+          implicit val costAccountingAlg = CostAccountingAlg.unsafe[Task](CostAccount.zero)
```

Comment:
Could you check if your new unit test passes without these changes to where `costAccountingAlg ` is defined? When I was refactoring the error handling I had asked you about this and I thought you told me the cost accounting is automatically reset on every call to `reducer.inj` or something.

Reply:
It is, the test will probably pass but with my upcoming changes to RHOL-11 (where we will actually make use of `phoLimit`) this will change because cost won't be set to 0 but will come from the deploy params.

Reply:
Right, makes sense.